modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -1594,12 +1594,51 @@ definition {
 	}
 
 	@description = "Verify that the user can publish a workflow with Timer configured"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanPublishWorkflowWithTimerConfigured {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanPublishWorkflowWithTimerConfigured";
+
+		Workflow.openWorkflowListView();
+
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
+
+		ProcessBuilderKaleoDesignerReact.dragSidebarNodeAndMove(
+			nodeLabel = "Task",
+			nodePosition = "-20,90",
+			nodeType = "task");
+
+		ProcessBuilderKaleoDesignerReact.viewAndFillTimers(
+			actionName = "Liferay Timer",
```

Comment:
The viewAndFillTimers needs to call [singleTimer = "true"] if it will be just one timer added.